### DISCUSSION TITLE
[Feature request] Automatically apply best-fit to VertiPaq Analyzer Columns

### DISCUSSION THREAD

**Discussion by otykier** [TabularEditor/TabularEditor3#1036] [Ideas]
### Discussed in https://github.com/TabularEditor/TabularEditor3/discussions/928

<div type='discussions-op-text'>

<sup>Originally posted by **corycundy** June 27, 2023</sup>
Every time I run Collect Stats on the VertiPaq Analyzer, the next thing I do is right click and select Best Fit (all columns). Could this be done automatically for each tab after the Collect Stats process is complete?</div>

**Comment by DBojsen** [comment]
Hi @corycundy

I have put this on our backlog.  No ETA yet.
